index.html

```diff
@@ -36,5 +36,147 @@ <h2>I’am a Software Developer</h2>
             </div>
         </div>
     </header>
+    <main>
+        <section class="works_head">
+            <div>
+                 <h3>My Recent Works</h3>
+                <hr>
+            </div>
+            <div>
+                <img src="img/workhead_card.png" alt="">
+            </div>
+            <div>
+                <h4>Multi-Post Stories</h4>
+                <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">html</button></li>
+                    <li><button type="button">bootstrap</button></li>
+                    <li><button type="button">Ruby</button></li>
+                </ul>
```

Comment:
As stated in the project requirement, Kindly implement the button interactions (enable, hover, etc.) 👍